sci-mathematics/glucas/patches/glucas-2.9.0.patchset

```diff
@@ -0,0 +1,21 @@
+From d618629166a8f96488b22685e99be986af57ab35 Mon Sep 17 00:00:00 2001
+From: nishanth1232 <thumma.nishanth79@gmail.com>
+Date: Fri, 16 Nov 2018 18:53:27 +0000
+Subject: gsetup.c for glucas
+
+
+diff --git a/gsetup.c b/gsetup.c
+index 7f32832..9a3d46c 100644
+--- a/gsetup.c
++++ b/gsetup.c
+@@ -39,7 +39,7 @@ void setup (int nonice)
+ #ifdef __sgi__
+     (void)schedctl(NDPRI, 0, NDPLOMIN);
+ #endif
+-#if !defined(macintosh) && !defined(pccompiler)
++#if !defined(macintosh) && !defined(pccompiler) !defined(_HAIKU_)
```

Comment:
```suggestion
+#if !defined(macintosh) && !defined(pccompiler) && !defined(_HAIKU_)
```

Reply:
@Begasus Still getting the same error after typing in:
```
#ifndef _HAIKU_
```
and:
```
&& !defined(_HAIKU_)
```

Reply:
See my comment: https://github.com/haikuports/haikuports/pull/3372/files#r234407192